app/submitter/converter.py

```diff
@@ -104,6 +105,7 @@ def convert_answers(
 def build_collection(metadata) -> Mapping[str, str]:
     return {
         "exercise_sid": metadata["collection_exercise_sid"],
+        "instrument_id": metadata.get("form_type"),
```

Comment:
Any reason why it is `get` ? whereas others are not.  is it possible to have `None`?

Reply:
It's not mandatory https://github.com/ONSdigital/eq-questionnaire-runner/blob/master/app/storage/metadata_parser.py#L104 son could be None

Reply:
url Link seems broken

Reply:
Metadata_parser got moved and merged in this PR(https://github.com/ONSdigital/eq-questionnaire-runner/pull/710/files) after I made that link. Its now https://github.com/ONSdigital/eq-questionnaire-runner/blob/master/app/utilities/metadata_parser.py#L104